src/_reactComponents/PanelHeaderComponents/IncrementMenu.jsx

```diff
@@ -6,6 +6,12 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 const FONT_SIZES = [8, 9, 10, 11, 12, 14, 18, 24, 30, 36, 48, 60, 72, 96];
```

Comment:
IncrementMenu has a dropdown menu that needs to be aligned below the textfield when there is a label, a vertical label, or no label